### ISSUE TITLE
Feature: Add an option to disable selection checkbox feature

### ISSUE THREAD

**Issue by hishitetsu** [files-community/Files#11938]
### What feature or improvement do you think would benefit Files?

Checkbox behavior changed in #11546. This eliminates the need for a dedicated checkbox column, but some users do not need the checkbox function itself. So there should be an option to disable selection checkbox feature.

### Requirements

- Add an option to disable selection checkbox feature
- Checkboxes do not appear when items are hovered or selected if disabled.

### Files Version

2.4.60.0

### Windows Version

Windows 11 22H2

**Comment by yaira2** [comment]
The only downside to the checkbox is that it covers up the thumbnail on hover (and only in the details layout). We want to give this change some time to settle and see if users get used to it before adding a setting to turn it off.

**Comment by lilacisfastest** [comment]
I second OP's thoughts. I use the details layout to work with a lot of image files, and use the thumbnails to quickly navigate my folders. While checkboxes are a great feature, they can sometimes get in the way. Perhaps they could be put next to the thumbnails instead of covering them, giving us the best of both worlds? Thanks!

**Comment by yaira2** [comment]
Either way there is a tradeoff, putting them next to the thumbnails takes up valuable horizontal space.

**Comment by takaokim** [comment]
I agree, it's annoying that the checkbox is activated by selecting a single item. It should fire only if you hold down the shift key or select multiple items with the mouse

**Comment by yaira2** [comment]
@hishitetsu should clicking the thumbnail still select the item?

**Comment by takaokim** [comment]
In my opinion, if you select the thumbnail it should select the element but not the checkbox (unless you specifically activate it with the mouse). But if you select several elements, then the checkbox of these should be activated. I think it would be the most intuitive way for the user.

**Comment by hishitetsu** [comment]
> @hishitetsu should clicking the thumbnail still select the item?

It should. It is not intuitive that clicking on a thumbnail does not select the item.

How about not showing checkboxes when no item is selected? Because there is no difference between clicking a checkbox and clicking a thumbnail until any item is selected. Checkboxes should appear when any item is selected. This way, there will be fewer comments about wanting to disable the checkboxes.

**Comment by yaira2** [comment]
Is it necessary to provide a third option to never show check boxes (even when multiple items are selected)?

**Comment by hishitetsu** [comment]
I don't think it is necessary. I'm uncomfortable with the empty checkboxes that appear just by hovering over an item. Simply not displaying this should greatly reduce discomfort.
![スクリーンショット 2023-04-04 092844](https://user-images.githubusercontent.com/66369541/229655751-dbb514a0-7cd7-4ce6-9541-9d768529c531.png)

If checkboxes appear after an item is selected, no functionality is lost.

**Comment by yaira2** [comment]
Looking at this from the other extreme, do we need a setting at all? Maybe we only start showing the checkbox when the user clicks a thumbnail.

**Comment by hishitetsu** [comment]
I agree. It would be good to start with this.

**Comment by hishitetsu** [comment]
May I open another feature request? I would prefer to keep this in case we should add an option to disable checkboxes.

**Comment by yaira2** [comment]
Of course

**Comment by Ajinkya2024** [comment]
hello folks,
I also believe that there should be an option to disable this function. As others have mentioned in detail view tickmark covers the thumbnail and it confuses while selecting multiple files of different formats. also in column view you don't get this tickmark when you select a file.
![Screenshot 2023-04-06 151638](https://user-images.githubusercontent.com/130047571/230346462-e1b1f228-8577-410b-a30b-a658bc44dae4.png)

**Comment by ArmandoRl1** [comment]
Yup I also think there should be an option to disable checkboxes.

**Comment by yaira2** [comment]
Thank you for your feedback, @hishitetsu implemented a change so that the check box only shows when the mouse is on top of the thumbnail.

**Comment by Ajinkya2024** [comment]
when to expect this new update?

**Comment by yaira2** [comment]
We're targeting the end of April but you can already try some of the improvements in the latest preview build. https://files.community/download

**Comment by Ajinkya2024** [comment]
adding an option to disable the checkbox feature would help. this is still there in the beta version. if I click on any file, that file thumbnail gets covered by a checkbox. Is there any option to go back to the previous version?

**Comment by yaira2** [comment]
The next preview build will have an option to hide the checkbox even when items are selected.
